activerecord/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
+*   Add `ActiveRecord::Base#previously_new_record?` to show if a record was new before the last save.
+
+    *Tom Ward*
+
 *   Allow generated `create_table` migrations to include or skip timestamps.
 
-     *Michael Duchemin*
+    *Michael Duchemin*
```

Comment:
Looks addressed by https://github.com/rails/rails/commit/cc27e9988f9ce4e52acf491a5f67be70ab7d25d8#diff-1ed2907b3b8f148c2533558a77673ffaR17 already, I think you can :scissors: this from this PR.